Makefile

```diff
@@ -90,13 +90,24 @@ helm-template:
 
 default: helm-template
 
+generate-s2i-docs:
+	./docs/scripts/generate-s2i-docs.sh
+
+clean-s2i-docs:
+	rm docs/s2i-*
```

Comment:
The `${RELEASE_DIR}` is meant to be a temporary directory, so there's no need to clean files -- which may accidentally delete valid files.